qa/pull-tester/rpc-tests.py

```diff
@@ -391,6 +391,10 @@ def runtests():
                     trimmed_tests_to_run.append(t)
             tests_to_run = trimmed_tests_to_run
 
+        # if all specified tests are disabled just quit
+        if len(tests_to_run) == 0:
+            quit()
```

Comment:
Nit: Supposedly [`sys.exit` is more correct way to exit](https://stackoverflow.com/a/19747562/1745500).